src/sql/parts/modelComponents/toolbarLayout.css

```diff
@@ -71,3 +71,9 @@
 	-o-transform: scale(1.272019649, 1.272019649);
 	transform: scale(1.272019649, 1.272019649);
 }
+
+.toolbarSeparator {
```

Comment:
Suggest making this

.modelview-toolbar-container .toolbar-separator

for both extra safety (so the style isn't accidently applied to other components outside the toolbar-container) and for following naming style of other styles.